modules/ROOT/pages/7.4-release-notes.adoc

```diff
@@ -160,6 +160,13 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Mouse hover on partially visible dialog collection elements no longer scrolls
+// #TINY-9915
+
+Previously, an issue caused partially visible collection elements, such as in the Character Map or Emojis dialog, to automatically scroll into view when hovered over.
```

Comment:
I would add that the scroll was unexpected and jumpy to not sound like we are removing a feature

Reply:
Updated based on the suggested improvement @ztomaszyk 
```suggestion
Previously, an issue caused partially visible collection elements, such as those in the Character Map or Emojis dialog, to unexpectedly scroll into view with abrupt and unintended movement when hovered over, resulting in a disruptive user experience.
```